pandas/core/strings.py

```diff
@@ -1962,11 +1934,269 @@ def cons_row(x):
                 cons = self._orig._constructor
                 return cons(result, name=name, index=index)
 
-    @copy(str_cat)
-    def cat(self, others=None, sep=None, na_rep=None):
-        data = self._orig if self._is_categorical else self._data
-        result = str_cat(data, others=others, sep=sep, na_rep=na_rep)
-        return self._wrap_result(result, use_codes=(not self._is_categorical))
+    def _str_cat_los(self, input, ignore_index=False):
```

Comment:
can you give this a better name, I have no idea what 'los' is (IOW make that part longer)

Reply:
was intended to be "List Of Series", but fixed